backend/danswer/connectors/confluence/connector.py

```diff
@@ -76,26 +80,31 @@ def _extract_confluence_keys_from_datacenter_url(wiki_url: str) -> tuple[str, st
         + parsed_url.path.split(DISPLAY)[0]
     )
     space = DISPLAY.join(parsed_url.path.split(DISPLAY)[1:]).split("/")[0]
-    return wiki_base, space
+    page_id = parsed_url.path.split("/")[5]
```

Comment:
I don't think this works in the general case, since afaik someone can put something like `https://danswer.ai/confluence/blah/blah/blah` as the wiki_base. In other words, I think we need to do the counting after `/display/`.

Also what happens if the page isn't included? This seems like it would break

Reply:
Also how (if at all) did we verify that this works for data center? Is there a link to a guide that we can put at the start of the func that shows the standard URL structure of datacenter deployments?

That this works correctly is quite important since many of our customers are datacenter users